pkg/service/user.go

```diff
@@ -972,7 +971,11 @@ func (s *userService) RetrieveCliConfig(ctx context.Context, req *userrpcv3.ApiK
 	// get the default project associated to this account
 	ap, err := dao.GetDefaultAccountProject(ctx, s.db, uuid.MustParse(req.Id))
 	if err != nil {
-		return nil, err
+		_log.Debug("unable to fetch account permissions for default project, looking further..")
+		ap, err = dao.GetAccountProjectByPermission(ctx, s.db, uuid.MustParse(req.Id), cliConfigR)
```

Comment:
What is the reason we are fetching account by permission here?

Reply:
to ensure it adheres to custom roles requirement.